cognite/client/data_classes/_base.py

```diff
@@ -297,6 +307,15 @@ def dump(self, camel_case: bool = True) -> list[dict[str, Any]]:
         """
         return [resource.dump(camel_case) for resource in self.data]
 
+    def dump_yaml(self, filepath: str | Path) -> None:
```

Comment:
An alternative could be to send in a stream handler like io.TextIOBase? So do: with open as f: dump_yaml(f)

Reply:
(or both)